piws/site_cubicweb.py

```diff
@@ -30,4 +30,26 @@
         "group": "piws",
         "level": 1,
     }),
+    ('enable-apache-deauth',
+     {'type' : 'string',
+      'default': 'no',
+      'help': 'Enable apache de-authentication',
+      'group': 'piws',
+      'level': 1,
+      }),
+    ('apache-cleanup-session-time',
+     {'type' : 'time',
+      'default': None,
+      'help': ('Duration of inactivity after which an apache de-authentication'
+               'will be triggered'),
+      'group': 'piws',
+      'level': 1,
+      }),
+    ('account-manager-url',
```

Comment:
This is the url where the user is redirected when his session has expired.
The parameter name is too specific since we may not use the account_manager:
de-authentification-redirection-url?